supernode/daemon/mgr/gc/gc_dfget_task.go

```diff
@@ -0,0 +1,63 @@
+package gc
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+func (gcm *Manager) gcDfgetTaskWithTaskID(ctx context.Context, taskID string, cids []string) bool {
+	errMap := make(map[string]error)
+	for _, cid := range cids {
+		err := gcm.dfgetTaskMgr.Delete(ctx, cid, taskID)
```

Comment:
yeah, the logic is totally correct. While I am wondering if we could do this:
```
for _, cid := range cids {
    if err := gcm.dfgetTaskMgr.Delete(ctx, cid, taskID); err != nil{
        errMap[cid] = err
    }
}
```

Reply:
We need to have a record which tasks failed to execute and why they failed. So I use a map to do that. 

And do you have any better ideas to do that? THX. 😄